torch/backends/cuda/__init__.py

```diff
@@ -94,7 +94,7 @@ def __init__(self):
         self.__initialized = True
 
     def __getitem__(self, device):
-        index = torch.cuda._utils._get_device_index(device)
+        index = torch._utils._get_device_index(device)
```

Comment:
As we discussed offline, we do not need to change it now.

Reply:
done.